reporter/src/funTest/kotlin/reporters/SpdxDocumentReporterTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.reporters
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.reporter.ReporterInput
+import org.ossreviewtoolkit.reporter.reporters.SpdxDocumentReporter.FileFormat
+import org.ossreviewtoolkit.spdx.SpdxModelSerializer
+import org.ossreviewtoolkit.spdx.model.SpdxDocument
+import org.ossreviewtoolkit.utils.ORT_NAME
+import org.ossreviewtoolkit.utils.test.patchExpectedResult
+import org.ossreviewtoolkit.utils.test.readOrtResult
+
+import java.io.File
+
+private val REPORT_OPTIONS = mapOf(
+    SpdxDocumentReporter.DOCUMENT_COMMENT to "some document comment",
+    SpdxDocumentReporter.DOCUMENT_NAME to "some document name",
+    SpdxDocumentReporter.CREATION_INFO_COMMENT to "some creation info comment"
+)
+
+class SpdxDocumentReporterTest : WordSpec({
+    "SpdxDocumentReporter" should {
+        "create the expected JSON SPDX document" {
+            val ortResult = readOrtResult("src/funTest/assets/static-html-reporter-test-input.yml")
+
+            val jsonSpdxDocument = generateReport(ortResult, FileFormat.JSON)
+
+            jsonSpdxDocument shouldBe patchExpectedResult(
+                File("src/funTest/assets/spdx-document-reporter-expected-output.spdx.json"),
+                "<REPLACE_UUID>" to jsonSpdxDocument.fromJson().uuid()
+            )
+        }
+
+        "create the expected YAML SPDX document" {
+            val ortResult = readOrtResult("src/funTest/assets/static-html-reporter-test-input.yml")
+
+            val yamlSpdxDocument = generateReport(ortResult, FileFormat.YAML)
+
+            yamlSpdxDocument shouldBe patchExpectedResult(
+                File("src/funTest/assets/spdx-document-reporter-expected-output.spdx.yml"),
+                "<REPLACE_UUID>" to yamlSpdxDocument.fromYaml().uuid()
+            )
+        }
+    }
+})
+
+private fun generateReport(ortResult: OrtResult, format: FileFormat): String {
+    val input = ReporterInput(ortResult)
+    val reportOptions = REPORT_OPTIONS + Pair("output.file.formats", format.toString())
```

Comment:
How about inlining `REPORT_OPTIONS` as it's only used here, and that would make the code to add the variable `Pair` a bit nicer IMO:

    val reportOptions = mapOf(
        SpdxDocumentReporter.DOCUMENT_COMMENT to "some document comment",
        SpdxDocumentReporter.DOCUMENT_NAME to "some document name",
        SpdxDocumentReporter.CREATION_INFO_COMMENT to "some creation info comment",
        "output.file.formats" to format.toString()
    )

Reply:
Done.